Watchdog/Watchdog.ino

```diff
@@ -27,34 +27,48 @@ void loop() {
     
     //NOT STARTED YET
     case 0:
```

Comment:
Please change from `case` to `if`. Usually `case` is only used for really short pieces of code (like selecting a value). Case statements just get hard to read